src/OrchardCore/OrchardCore.ResourceManagement/ServiceCollectionExtensions.cs

```diff
@@ -14,4 +15,14 @@ public static IServiceCollection AddResourceManagement(this IServiceCollection s
 
         return services;
     }
+
+    /// <summary>
+    /// Adds a service implementing <see cref="IConfigureOptions{ResourceManagementOptions}"/> to the service collection.
+    /// </summary>
+    /// <typeparam name="T">
+    /// The type of the service implementing <see cref="IConfigureOptions{ResourceManagementOptions}"/> to.
+    /// </typeparam>
+    public static IServiceCollection AddResourceManagementOptionsConfiguration<T>(this IServiceCollection services)
```

Comment:
Seb hates @MikeAlhayek for introducing such extensions, seems it's your turn you :)